### PR TITLE
lora: add lora and loramac config macros to config doc group

### PR DESCRIPTION
**Pull request by jia200x**
### Contribution description
See #10566 

I also moved some `@brief` tags outside of the `#ifndef` block for consistency, and fixed a minor issue with an extra `<` character in a doxygen tag.

### Testing procedure
Compile documentation. Two new group for the configuration macros of `net_lora` and `net_loramac`should show up both in the `config` group doc as well in the doc of the respective `net_lora` and `net_loramac` (sub-)modules changed here.

### Issues/PRs references
Addresses #10566 in part

### CODE REVIEW

File: sys/include/net/lora.h [hunk 1]
```diff
@@ -29,61 +29,75 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRa modulation default values
+ * @defgroup net_lora_conf    LoRa modulation default values
```
Comment (miri64) [inline sys/include/net/lora.h]: Unifying with the other configuration groups

```suggestion
 * @defgroup net_lora_conf     LoRa modulation compile configurations
```

?

File: sys/include/net/loramac.h [hunk 2]
```diff
@@ -30,7 +30,8 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRaMAC default parameters
+ * @defgroup net_loramac_conf    LoRaMAC default parameters
```
Comment (miri64) [inline sys/include/net/loramac.h]: Unifying with the other configuration groups

```suggestion
 * @defgroup net_loramac_conf     LoRaMAC compile configurations
```

?

File: sys/include/net/lora.h (hunk 1 above)
Comment (jia200x) [reply sys/include/net/lora.h]: roger

File: sys/include/net/loramac.h (hunk 2 above)
Comment (jia200x) [reply sys/include/net/loramac.h]: roger

File: sys/include/net/lora.h [hunk 3]
```diff
@@ -29,61 +29,75 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRa modulation default values
+ * @defgroup net_lora_conf     LoRa modulation compile configurations
```
Comment (miri64) [inline sys/include/net/lora.h]: Very nit-picky

```suggestion
 * @defgroup net_lora_conf  LoRa modulation compile configurations
```

File: sys/include/net/loramac.h [hunk 4]
```diff
@@ -30,7 +30,8 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRaMAC default parameters
+ * @defgroup net_loramac_conf     LoRaMAC compile configurations
```
Comment (miri64) [inline sys/include/net/loramac.h]: Also very nit-picky.

```suggestion
 * @defgroup net_loramac_conf   LoRaMAC compile configurations
```

**Comment by miri64** [comment]
Please squash

**Pull request merged**